test/github-issues/3857/entity/Women.ts

```diff
@@ -0,0 +1,9 @@
+import {Person} from "./Person";
+import {ChildEntity, Column} from "../../../../src";
+
+@ChildEntity()
+export class Women extends Person {
+
+    @Column("int")
+    brassiereSize: number;
```

Comment:
haha dude you need to spend less time on a programming and more time with women 🤣

Reply:
OK, I'll tell my girlfriend, that she sould let me go out more often ;)